nullaway/src/main/java/com/uber/nullaway/fixserialization/out/SuggestedNullableFixInfo.java

```diff
@@ -26,64 +26,56 @@
 import com.sun.source.util.TreePath;
 import com.uber.nullaway.ErrorMessage;
 import com.uber.nullaway.fixserialization.location.FixLocation;
-import com.uber.nullaway.fixserialization.qual.AnnotationConfig;
 import java.util.Objects;
 
-/** Stores information suggesting a type change of an element in source code. */
-public class SuggestedFixInfo {
+/** Stores information suggesting adding @Nullable on an element in source code. */
+public class SuggestedNullableFixInfo {
```

Comment:
Why the rename here? It makes sense for now, but I wonder if we will have to flip it right back once we start adding other types of fixes? Or maybe the idea will be to add a common supertype for fixes and retain this one as a subclass?

Reply:
I did the renaming since instantiating this class only means we are suggesting `Nullable` and the value `Nullable` is hardcoded in this class. I though for better understanding the class purpose, I should refine the name.

Reply:
Sounds good for now. We might revisit this if we are adding other kinds of fixes, but right not it isn't clear if that will happen within NullAway or within the annotator or where, so this makes sense to me.